templates/config.json

```diff
@@ -256,6 +264,13 @@
       "id": "access",
       "file": "policies/access.md",
       "name": "Access",
+      "instructions": [
+        "What are your primary critical systems and IdP?",
+        "Do unused accounts, passwords, and access keys get removed? How frequently?",
+        "How frequently is systems access reviewed?",
+        "How soon after termination is system access revoked?",
+        "Is your organization subject to HIPAA compliance?"
```

Comment:
I don't think these instructions/questions are applicable at the Policy level. They seem to be asking very specific control procedural questions. 

Policies are high level descriptions of what an organization promises to do. Control Procedures are the what and how each policy is being implemented.

The access policy template we have can be simplified a bit, with certain details moved to the controls and procedures.

Reply:
Thought from the team: So [this Policy](https://j1dev.apps.us.jupiterone.io/policies/access/cp-access-password) has mentions of a specific expiration date, and HIPAA and other things in the policy related to the above questions. That being said, these should all be asked again in the control-procedure. Can we edit the Policy templates to remove the specific details of expiry, etc. so the details are only at the procedure level? George happy to do it :)

Reply:
Removed instructions from policy